dashboard/src/components/relocation/Toast.js

```diff
@@ -208,12 +208,13 @@ class Toast extends React.PureComponent {
               onEnd={onClose}
               paused={mouseOver}
             >
-              {showAgeIndicator &&
-                (progress => (
-                  <CircularProgress width={4} value={progress} opacity={0.5}>
-                    {closeButton}
-                  </CircularProgress>
-                ))}
+              {showAgeIndicator
+                ? progress => (
+                    <CircularProgress width={4} value={progress} opacity={0.5}>
+                      {closeButton}
+                    </CircularProgress>
+                  )
+                : null}
```

Comment:
Not sure if this is the best way to handle it, but Timer wants a child to be a function, previously I was getting console errors about it being a boolean. This was just a really quick change.

Reply:
👍